packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html

```diff
@@ -216,6 +216,7 @@ <h2 data-translate="refunds_title2"></h2>
         </tbody>
     </table>
     <div
+        data-ng-show="refunds.list.results.length > 0"
```

Comment:
```suggestion
        data-ng-if="refunds.list.results.length > 0"
```

Reply:
don't think ng-if will work here as pagination-server-side is still required to initialize and fetch data
ng-show seems appropriate

Reply:
it break the display with this method :/

Reply:
yes the `div` must be in the dom in order to instanciate the directive on it. So `ngIf` won't work